src/Management/src/Endpoint/Mappings/AspNetCoreRouteDetails.cs

```diff
@@ -13,4 +13,8 @@ public class AspNetCoreRouteDetails : IRouteDetails
     public IList<string> Produces { get; set; }
 
     public IList<string> Consumes { get; set; }
+
+    public IList<string> Params { get; set; }
+
+    public IList<string> Headers { get; set; }
```

Comment:
Note I'm going to initialize these properties with empty collections when merging this into #1155, based on what we discussed yesterday. I already did that for all existing collection properties there, so I think it makes the most sense to handle that in my PR.